resources/clients/react/elwing/src/plugins/sqs-message/package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "sqs-chat",
```

Comment:
Changes in this file look unrelated to the gitignore changes, were these intentional to include?

Reply:
I ran an npm install to re-generate some node modules, this updated as a result. It's not directly related, but it's a valid change.

Reply:
ok.